### ISSUE TITLE
Cron or something else? "Emails to send" are permanently waiting.

### ISSUE THREAD

**Issue by dannypingpong** [YetiForceCompany/YetiForceCRM#6127]
#### Issue
"Emails to send" are piling up. All showing status "waiting to be sent". 

#### Actual Behavior
No workflow emails are automatically sent out unless I click on the blue button of "manual sending". Then emails are sent with no problem. This tells me the SMTP setting is working. 

#### Expected Behavior
I set up email notification when "assign to" has change in Project module, I also updated the cron settings to run "workflow" every 1 minute, so I expected the notification emails to be sent within 1 minute after the project record is changed. 

#### How to trigger the error

http://jkonline.ca/danny/no-email-sent.gif

From my settings my understanding is that the cron (found in settings --> automation tab --> cron) runs workflow every minute. But the webpage tells me it has no "last scan started/ended" so to me it has never run. 

Do I need to specifically add a cron job in my hosting cpanel (A2hosting) to purposely tell the hosting server run a cron job every single minute to pick up the workflow?

Sorry if I asked a stupid question... Thanks.

#### Your Environment

* YetiForce Version used: 4.3
* Browser name and version: Chrome the latest

**Comment by vovpff** [comment]
@dannypingpong workflow hooks your changes if email added. Check email sending cron task interval.

**Comment by dannypingpong** [comment]
@vovpff Thanks for your reply! 

Yes I did check the email sender job, it's active and set to 1min internal, so that shouldn't be the cause to the problem. 
jkonline.ca/danny/cron-question-1.png

My question is now... do I need to manually add a cron job to run some yetiforce-cron.pl or yetiforce-cron.sh (I just made up the file name...) in the cPanel? 
http://jkonline.ca/danny/cron-question-2.png

If so can someone please kindly let me know if there is a centralized cron script for Yeti?

Thanks!

**Comment by dannypingpong** [comment]
Sorry, it is my stupidity. I found this:
https://oldwww.yetiforce.com/en/administrator-documentation/panel/cron/150-enable-cron.html

and I followed the procedure to have a Cron job added in my hosting's cPanel. It ran to error, none of the workflows worked, but at least I got something and I know it is something wrong in the vtigercron.sh that is stopping the workflows from running. I will open up another ticket if I cannot get a good fix after some researches.
